openprescribing/frontend/tests/test_views.py

```diff
@@ -453,3 +454,18 @@ def test_price_per_unit_histogram_with_practice(self):
             'highlight': ['P87629'],
             'date': ['2014-11-01'],
         })
+
+
+class TestGetMeasureTagFilter(TransactionTestCase):
+
+    def test_rejects_bad_tags(self):
+        with self.assertRaises(BadRequestError):
+            _get_measure_tag_filter({'tags': 'nosuchtag'})
+
+    def test_filters_on_core_tag_by_default(self):
+        tag_filter = _get_measure_tag_filter({})
+        self.assertEqual(tag_filter['tags'], ['core'])
+
+    def test_returns_tag_name(self):
+        tag_filter = _get_measure_tag_filter({'tags': 'lowpriority'})
+        self.assertEqual(tag_filter['name'], 'Low Priority')
```

Comment:
Could you add tests for the `show_all_by_default` flag?

Reply:
And the `show_message` key in the return value?

Reply:
Yes, of course. Sorry this PR is horribly light on tests at the moment. I thought it would be better to get eyes on it and feedback before I spent too much time writing tests. (I'm am very much not a TDD person I'm afraid -- much as I would like would to be :)